bench-tps/src/bench_tps_client/high_tps_client.rs

```diff
@@ -0,0 +1,233 @@
+//! Client optimized for bench-tps: high mean TPS and moderate std
+//! This is achieved by:
+//! * optimizing the size of send batch
+//! * sending batches using connection cache in nonblocking fashion
+//! * use leader cache from TpuClient to minimize rpc calls
+//! * using several staked connection caches (not implemented yet)
+//!
+use {
+    crate::bench_tps_client::{BenchTpsClient, BenchTpsError, Result},
+    solana_client::{
+        connection_cache::Protocol, nonblocking::tpu_client::LeaderTpuService,
```

Comment:
can you import this directly from `solana_tpu_client::nonblocking::tpu_client::LeaderTpuService`? sounds like we're eventually going to get rid of the `solana_client` according to Tyera